aws/resource_aws_mwaa_environment.go

```diff
@@ -0,0 +1,753 @@
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(false),
+						},
+						"scheduler_logs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(false),
+						},
+						"task_logs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(true),
+						},
+						"webserver_logs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(false),
+						},
+						"worker_logs": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem:     mwaaEnvironmentModuleLoggingConfigurationSchema(false),
+						},
+					},
+				},
+			},
+			"max_workers": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      10,
+				ValidateFunc: validation.IntAtLeast(1),
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"network_configuration": {
+				Type:     schema.TypeList,
+				Required: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"security_group_ids": {
+							Type:     schema.TypeSet,
+							Required: true,
+							MinItems: 1,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+						"subnet_ids": {
+							Type:     schema.TypeSet,
+							Required: true,
+							ForceNew: true,
+							MinItems: 2,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+					},
+				},
+			},
+			"plugins_s3_object_version": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"plugins_s3_path": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"requirements_s3_object_version": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"requirements_s3_path": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"service_role_arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"source_bucket_arn": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validateArn,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"webserver_access_mode": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  mwaa.WebserverAccessModePrivateOnly,
+				ValidateFunc: validation.StringInSlice([]string{
+					mwaa.WebserverAccessModePrivateOnly,
+					mwaa.WebserverAccessModePublicOnly,
+				}, false),
+			},
+			"webserver_url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"weekly_maintenance_window_start": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"tags": tagsSchema(),
```

Comment:
Keep alphabetical ordering: #16763. Move to between `status` and `webserver_access_mode`.